python/core/unified_api_server.py

```
@@ -7996,6 +8200,282 @@ def _get_mock_analysis_results() -> List[GameAnalysisSummary]:
+    """
+    if not usage_tracker:
+        raise HTTPException(status_code=503, detail="Usage tracking not configured")
+
+    user_id = request.get('user_id')
+
+    if not user_id:
+        raise HTTPException(status_code=400, detail="user_id is required")
+
+    try:
+        stats = await usage_tracker.get_usage_stats(user_id)
+        return stats
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)